proof-of-concepts/aws-scalable-app-proxy/instance.tf

```diff
@@ -0,0 +1,128 @@
+locals {
+  user_data = <<USER_DATA
+#!/usr/bin/env bash
+set -e
+exec >> /var/log/user_data.log 2>&1
+
+source /etc/os-release
+
+apt-get update
+
+apt-get install -y curl gnupg2 ca-certificates apt-transport-https jq awscli
+
+# Get AWS credentials for interacting with AWS resources
+#TOKEN=$(curl -fsSL -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")
+#REGION=$(curl -fsSL -H "X-aws-ec2-metadata-token: $TOKEN" http://169.254.169.254/latest/dynamic/instance-identity/document | jq -r '.region')
+#ROLE=$(curl -fsSL -H "X-aws-ec2-metadata-token: $TOKEN" http://169.254.169.254/latest/meta-data/iam/security-credentials)
+#CREDS=$(curl -fsSL -H "X-aws-ec2-metadata-token: $TOKEN" http://169.254.169.254/latest/meta-data/iam/security-credentials/$ROLE)
+#
+#export AWS_ACCESS_KEY_ID=$(echo $CREDS | jq -r '.AccessKeyId')
+#export AWS_SECRET_ACCESS_KEY=$(echo $CREDS | jq -r '.SecretAccessKey')
+#export AWS_SESSION_TOKEN=$(echo $CREDS | jq -r '.Token')
+#export AWS_DEFAULT_REGION=$REGION
```

Comment:
```suggestion
```

Would suggest this whole commented section can just be removed as it's not used.